release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -13,6 +13,8 @@ import '../services/conductor.dart';
 import '../state/status_state.dart';
 import 'common/tooltip.dart';
 
+/// Very important! The order of this enum decides which order the initialization
```

Comment:
You can drop the `Very important!` as this being the first sentence indicates its importance